docker/docker-compose.yml

```diff
@@ -118,6 +118,30 @@ services:
       - 'traefik.http.middlewares.strip_feed_discovery_prefix.stripprefix.forceSlash=true'
       - 'traefik.http.routers.feed-discovery.middlewares=strip_feed_discovery_prefix'
 
+  # user service
+  user:
+    container_name: 'user'
```

Comment:
users throughout this definition (also below in traefik)

Reply:
`context: ../src/api/user`:  should I also rename the entire micro-service folder to `users` then?